acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/util/BitbucketUtils.java

```diff
@@ -0,0 +1,242 @@
+package it.com.atlassian.bitbucket.jenkins.internal.util;
```

Comment:
Ah yes, that's because this class is (sadly) a duplicate of its counterpart in the main module. Those unused methods _may_ be useful in future tests, but I suppose we can add them when we need them. :+1: 

- [ ] delete unused stuff